index.bs

```diff
@@ -211,6 +212,16 @@ Cancelling the socket's ReadableStream and closing the socket's WritableStream d
 
 The {{upgraded}} attribute is a boolean flag that indicates whether the socket has been upgraded to a secure connection (using `startTLS()`).
 
+<h4 id="secureTransport-attribute">secureTransport</h4>
+
+The {{secureTransport}} attribute is a string value indicating the type of secure transport used by the socket. Possible values are:
+
+<ul>
+  <li>"on" - TLS/SSL connection</li>
+  <li>"starttls" - Opportunistic TLS connection</li>
+  <li>"off" - Insecure TLS connection</li>
+</ul>
```

Comment:
You'll need to update the `connect(...)` and `startTls(...)` algorithm definitions to include setting this attribute value.